src/components/SaveFilterTabDialog/SaveFilterTabDialog.tsx

```diff
@@ -1,12 +1,12 @@
 import {
-  Button,
   Dialog,
   DialogActions,
   DialogContent,
   DialogTitle,
   TextField
 } from "@material-ui/core";
 import { buttonMessages } from "@saleor/intl";
+import { Button } from "@saleor/macaw-ui";
```

Comment:
Same as above, import changes but the component props remain the same - make sure it's intentional